resync/workflows/nodes_verbose.py

```
@@ -0,0 +1,84 @@
+import structlog
+from typing import Any
+from sqlalchemy import text
+
+logger = structlog.get_logger(__name__)
+
+_JOB_EXECUTION_HISTORY_LIMIT = 2000
+_METRICS_HISTORY_LIMIT = 5000
+
+async def fetch_job_history(db: Any, job_name: str, cutoff_date: Any) -> list[dict[str, Any]]:
```

Ruff (0.15.0)

[warning] Unused function argument: `db`

(ARG001)